Resources/Maps/saltern.yml

```diff
@@ -72527,4 +72872,87 @@ entities:
   - pos: 55.5,-15.5
     parent: 853
     type: Transform
+- uid: 7104
+  type: FireExtinguisher
+  components:
+  - parent: 4944
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7105
+  type: FireExtinguisher
+  components:
+  - parent: 4945
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7106
+  type: FireExtinguisher
+  components:
+  - parent: 4946
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7107
+  type: FireExtinguisher
+  components:
+  - parent: 4947
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7108
+  type: FireExtinguisher
+  components:
+  - parent: 4948
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7109
+  type: FireExtinguisher
+  components:
+  - parent: 4949
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7110
+  type: FireExtinguisher
+  components:
+  - parent: 4950
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7111
+  type: FireExtinguisher
+  components:
+  - parent: 4951
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7112
+  type: FireExtinguisher
+  components:
+  - parent: 4952
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7113
+  type: FireExtinguisher
+  components:
+  - parent: 5335
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7114
+  type: FireExtinguisher
+  components:
+  - parent: 5336
+    type: Transform
+  - canCollide: False
+    type: Physics
+- uid: 7115
+  type: SpawnPointChemist
+  components:
+  - pos: 14.5,-0.5
+    parent: 853
+    type: Transform
```

Comment:
hard to tell but this makes it seem like mapping was fucked up and the map wasn't properly paused?

Reply:
Yea that isn't right, this looks fucked. Good catch.

Reply:
Hmm, that's strange because I never unpaused and followed the instructions on the mapping documentation exactly.

Reply:
https://github.com/space-wizards/space-station-14/pull/5501

Reply:
This was my fault. Was spawning starting entities into item slots on component startup instead of map init